types/react/test/elementAttributes.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const testCases = [
+    <span />,
+    <span accessKey="s" />,
+    <span className="klass" />,
+    <span contentEditable />,
+    <span contextMenu="menuId" />,
+    <span dir="rtl" />,
+    <span draggable />,
+    <span hidden />,
+    <span id="s" />,
+    <span lang="art-x-tokipona" />,
+    <input placeholder="placeholder" />,
+    <span slot="my-text" />,
+    <span spellCheck />,
+    <span tabIndex={0} />,
+    <span title="title" />,
+    <span role="button" />,
+    <span role="button" />,
```

Comment:
nit: duplicated

Reply:
Thanks! Fixed and supplemented with more test cases.